static/animation-iframes/approach_icon_1/approach_icon_1.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="chrome=1,IE=edge" />
+	<title>our_approach_animation</title>
+	<style>
+		html {
+			height:100%;
+		}
+		body {
+			margin:0;
+			height:100%;
+		}
+	</style>
+	<!-- copy these lines to your document head: -->
```

Comment:
Should these be here ye?

Reply:
Yeah these are the animations that gianluca exports. They are HTML pages they are then included using an component `HypeAnimation` which is a wrapper for the Iframe.